xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-server/xwiki-platform-search-solr-server-data/src/main/resources/xwiki/conf/schema.xml

```diff
@@ -242,7 +242,7 @@
    <dynamicField name="*_ar"  type="text_ar"    indexed="true"  stored="true" multiValued="true" />
    <dynamicField name="*_bg"  type="text_bg"    indexed="true"  stored="true" multiValued="true" />
    <dynamicField name="*_ca"  type="text_ca"    indexed="true"  stored="true" multiValued="true" />
-   <dynamicField name="*_cz"  type="text_cz"    indexed="true"  stored="true" multiValued="true" />
+   <dynamicField name="*_cs"  type="text_cz"    indexed="true"  stored="true" multiValued="true" />
```

Comment:
My bet here is that a simple confusion was made on the Solr part between country code and language code, as you`ve mentioned.

I understand your suggestion, but my opinion is that we avoid confusions altogether (for anyone reading the config), and possibly add a comment (above the type definition) for whoever will be doing the upgrade (us, most likely) that, until Solr fixes the problem on their part, for XWiki, text_cs means Solr's text_cz :)